patches/react-native+0.75.2+021+allow-recursive-commits-with-synchronous-mount-on-android.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native/ReactAndroid/src/main/jni/react/fabric/Binding.cpp b/node_modules/react-native/ReactAndroid/src/main/jni/react/fabric/Binding.cpp
+index 280ae46..2f29b9b 100644
+--- a/node_modules/react-native/ReactAndroid/src/main/jni/react/fabric/Binding.cpp
++++ b/node_modules/react-native/ReactAndroid/src/main/jni/react/fabric/Binding.cpp
+@@ -483,8 +483,13 @@ void Binding::schedulerShouldRenderTransactions(
+     return;
+   }
+ 
+-  if (ReactNativeFeatureFlags::
+-          allowRecursiveCommitsWithSynchronousMountOnAndroid()) {
++  // if (ReactNativeFeatureFlags::
++  //         allowRecursiveCommitsWithSynchronousMountOnAndroid()) {
```

Comment:
I think return true in `allowRecursiveCommitsWithSynchronousMountOnAndroid` declaration inside `ReactNativeFeatureFlags` is better. WDYT?

Reply:
The default value seems to be defined in four places, and this is the only place that reads this flag, so patching it here is easier. I'm also fine with changing the four default values. Do you prefer that?

https://github.com/facebook/react-native/blob/2e994fdddb05955ee8dcce1e2151288977dd1c5e/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/internal/featureflags/ReactNativeFeatureFlagsDefaults.kt#L30

https://github.com/facebook/react-native/blob/d1c72016c12b552b1140652209fb816a9facf0f2/packages/react-native/ReactCommon/react/featureflags/ReactNativeFeatureFlagsDefaults.h#L39

https://github.com/facebook/react-native/blob/d1c72016c12b552b1140652209fb816a9facf0f2/packages/react-native/scripts/featureflags/ReactNativeFeatureFlags.config.js#L48

https://github.com/facebook/react-native/blob/d1c72016c12b552b1140652209fb816a9facf0f2/packages/react-native/src/private/featureflags/ReactNativeFeatureFlags.js#L117

Reply:
Hmm! Ok, keep it as it is.

Reply:
I think no need for comments in L11, L12, We can remove it.

Reply:
Removed.